javascript/react-js/hooks.md

```diff
@@ -157,4 +163,12 @@ This section contains helpful links to other content. It isn't required, so cons
 
 * [Here](https://www.ohansemmanuel.com/react-hooks-documentation-easy-to-read/) is another article about hooks, which provides a simpler version of the official React hooks documentation.
 * [Watch this video](https://youtu.be/-MlNBTSg_Ww) for a look into `useState()`, `useEffect()` and custom hooks in great detail.
-  * There is currently a small bug in this program as it is currently written, which can be fixed by changing the url on CharPicker.js line 10 from https://swapi.co to https://swapi.dev.
\ No newline at end of file
+  * There is currently a small bug in this program as it is currently written, which can be fixed by changing the url on CharPicker.js line 10 from https://swapi.co to https://swapi.dev.
+
+  ### Knowledge Checks
```

Comment:
Looks like we've got an indent here that I don't *think* we want

Reply:
You're right, no indent needed here. I'll remove.